### DISCUSSION TITLE
Will Nylo be able to keep up with any future update from Flutter

### DISCUSSION THREAD

**Discussion by HynRidge** [nylo-core/nylo#128] [General] [answered]
Hi maintainer, I have a few questions regarding this library / micro-framework.

I am wondering if this library would be able to keep up with rapid changes of its dependent library? If yes, how would you guarantee this?

Additionally, I am looking to contribute to this project in own time as well. May I know if this project is an open-source project?

Best Regards,
HynRidge

**Comment by agordn52** [comment, marked as answer]
Hi @HynRidge,

No problem, I can answer these questions.

1. Yes, Nylo is regularly updated. You can check our releases here https://github.com/nylo-core/nylo/releases.

2. Yes, Nylo is open-source.

Thanks,
Anthony

**Reply by HynRidge** [reply]
Thanks for the swift answer @agordn52

**Reply by HynRidge** [reply]
Just a sidetrack, may I know if NyRouter support Deeplink and Applink? Thanks

**Reply by agordn52** [reply]
Hi @HynRidge,

No problem and yes. Nylo does support Deep linking.
If you follow this guide https://docs.flutter.dev/cookbook/navigation/set-up-universal-links#2-adjust-ios-build-settings it'll work. You can skip section 1.

**Reply by HynRidge** [reply]
Great! That's really awesome!

**Reply by HynRidge** [reply]
Also, I saw in your documentation that you have a State Management. However, I realized this is only works for a single stateful widget. If I want to maintain a global / app state management, may I know if Nylo support this? For example, I want to store the user preference value in the app state management for subsequent access to my app.

**Reply by agordn52** [reply]
Hi @HynRidge,

Check out this video on State management in Nylo https://www.youtube.com/watch?v=X5EVh1KooFk&pp=ygUdbnlsbyBmbHV0dGVyIHN0YXRlIG1hbmFnZW1lbnQ%3D

And yes, you can store data using `NyStorage` 👍

**Reply by HynRidge** [reply]
Hi, @agordn52. Love to hear that! Thank you so much!
